### PR TITLE
Implement "reform-db init" command

### LINKED ISSUE #13: "reform-db init" command
`reform-db init models` should connect to a database, inspect `information_schema` or equivalent, and generate model files from it.

See also #75 and #102.

See PR #113.

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #13.

TODO:
- [x] Check/clarify/document logic for SQLite3 types.
- [x] Add correct tail condition for MS SQL.
- [x] Check `time` import.
- [x] Check package name.
- [x] Document CLI.
- [x] Add `gofmt`.
- [x] Check/fix gometalinter issues.

### CODE REVIEW

File: reform-db/cmd_init.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+	var structs []StructData
+	switch db.Dialect {
+	case postgresql.Dialect:
+		structs = initModelsInformationSchema(db, `WHERE table_schema = current_schema()`, goTypePostgres)
+	case mysql.Dialect:
+		structs = initModelsInformationSchema(db, `WHERE table_schema = DATABASE()`, goTypeMySQL)
+	case sqlite3.Dialect:
+		structs = initModelsSQLite3(db)
+	case mssql.Dialect:
+		structs = initModelsInformationSchema(db, ``, goTypeMSSQL)
```
Comment (AlekSi) [inline reform-db/cmd_init.go]: @AlekseyMartynov What condition should be used here?

Comment (AlekseyMartynov) [inline reform-db/cmd_init.go]: Try this:
```sql
select * from information_schema.tables where table_catalog = db_name()
```

Comment (AlekseyMartynov) [inline reform-db/cmd_init.go]: P.S. and probably `and table_schema = schema_name()` for extra accuracy

Comment (AlekSi) [inline reform-db/cmd_init.go]: I've connected to database XXX, can I get information about tables from another database with `SELECT * FROM information_schema.tables`? I don't think so. `table_schema = schema_name()` should be enough, right?

**Comment by AlekSi** [comment]
@AlekseyMartynov @rumyantseva PTAL if you have some time.

File: reform-db/cmd_init.go (hunk 1 above)
Comment (AlekseyMartynov) [inline reform-db/cmd_init.go]: You are right. Checked and confirm that unqualified `information_schema` returns info for the current db only. `table_schema` is enough.

**Pull request merged**